pinot-spi/src/test/java/org/apache/pinot/spi/filesystem/LocalPinotFSTest.java

```diff
@@ -205,41 +205,12 @@ public void testFS()
       // Expected.
     }
 
-    // Check that directory only copy worked
```

Comment:
Can we add the test for `copyFromLocalDir()`?

Reply:
It is covered in the hadoopFs test -- that is what broke before.